src/inc/schedule-output-functions.php

```diff
@@ -263,6 +263,58 @@ function wpcs_schedule( $atts, $content ) {
 	return $return;
 }
 
+/**
+ * Get tags for a session.
+ *
+ * @param int    $talk_id Post ID for session.
+ * @param string $tag_heading_txt Heading text for tags.
```

Comment:
This parameter isn't used in the function; either incorporate in the function or remove doc.